sdk/identity/Azure.Identity/TROUBLESHOOTING.md

```diff
@@ -216,8 +216,7 @@ az account get-access-token --output json --resource https://management.core.win
 ```
 >Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
 
-<a name="TroubleshootAzurePowerShellCredential"></a>
-## Troubleshooting `AzurePowershellCredential` Authentication Issues
+## Troubleshoot `AzurePowerShellCredential` Authentication Issues
```

Comment:
Nit: did you check that the backticks don't mess up the stub?

Reply:
They don't affect it. This is the pattern used for all the other headers currently. I just missed updating it when I changed `Troubleshooting` to `Troubleshoot`.